lib/bibliotheca/exercise.rb

```diff
@@ -18,19 +18,31 @@ def text?
     end
 
     def process_choices!
-      choices_to_test! if choice? && text?
+      if text?
+        multiple_choices_to_test! if multiple_choice?
```

Comment:
Actually, when `multiple_choice?` is  `true`, `text?` is `true`, too. So there is some redundant logic here.